docs/core/basics_de.md

```diff
@@ -204,6 +204,19 @@ python3 -m http.server 8080 --directory /resources/web
 <img src="./img/dogu-in-ces.svg">
 
 ### Das Dogu testen
-- Ausgaben im Log prüfen, ob Dogu korrekt gestartet wurde:
-  - `cat /var/log/docker/newdogu.log`
-- `https://192.168.56.2/newdogu` aufrufen.
\ No newline at end of file
+
+Die Syslog-Konfiguration des Cloudogu EcoSystems sorgt dafür, dass jedes Dogu ein einzelnes Logfile unter dem Pfad
+`/var/log/docker/` schreibt, um bspw. Log-Forwarding oder Analysen zu ermöglichen.
+
+In diesem Log können nun Ausgaben geprüft werden, ob das Dogu korrekt gestartet wurde:
+- `cat /var/log/docker/newdogu.log`
+
+Alternativ kann auch der HTTP-Endpunkt des Dogus aufgerufen werden:
+- `https://192.168.56.2/newdogu` aufrufen.
+
+## Architektursicht eines integrierten Dogus
+
+Das folgende Schaubild zeigt mögliche Kommunikationswege eines integrierten Dogus (Redmine) im Cloudogu EcoSystem.
+Im Gegensatz zu dem minimalen Dogu aus dem vorherigen Kapitel besitzt Redmine andere Dogus als Abhängigkeiten, verwendet Service-Accounts und Benutzerinformationen über das CAS-Dogu und wird über den ETCD konfiguriert.
```

Comment:
I'd like to emphasize that now more than ever, the existance of ETCD as local registry is not as fixed as we used to think. I suggest to replace ETCD with a more abstract term.

```suggestion
Im Gegensatz zu dem minimalen Dogu aus dem vorherigen Kapitel besitzt Redmine andere Dogus als Abhängigkeiten, verwendet Service-Accounts und Benutzerinformationen über das CAS-Dogu und wird über die EcoSystem-Registry konfiguriert.
```